utils/copy_ami_to_all_regions.sh

```diff
@@ -0,0 +1,10 @@
+AMI_ID=ami-00c1a16b5136fbb7c
+AMI_NAME='scylla-qa-loader-ami-v19'
+SOURCE_REGION=us-east-1
+TARGET_REGIONS='us-west-2 eu-west-1 eu-west-1 eu-west-2 eu-central-1'
```

Comment:
The `eu-west-1` one is duplicated twice and `eu-north-1` is missing.

Reply:
fixed